addons/func_godot/src/map/func_godot_map.gd

```diff
@@ -82,6 +95,11 @@ func verify_and_build() -> void:
 	else:
 		emit_signal("build_failed")
 
+func _on_reimport(resources: PackedStringArray) -> void:
+	for resource in resources:
```

Comment:
An alternative is to connect it in our ready block, and do an autobuild guard check here.
```
if !auto_build:
    return;
```
That said, each reimport would still hit this function.

Reply:
Ah, that's a good point. It would write cleaner that way, rather than fiddling with signals all over.